tutorials/3d/introduction_to_3d.rst

```diff
@@ -11,11 +11,213 @@ are present in both 2D and 3D versions. In fact, it is worth checking
+**right-handed** coordinate system. This means that for most objects that need
+alignment (such as lights or cameras), the Z axis is used as a "pointing
+towards" direction. This convention roughly means that:
+
+-  **X** is sides
+-  **Y** is up/down
+-  **Z** is front/back
+
+See this chart for comparison with other 3D software:
+
+.. figure:: img/introduction_to_3d_coordinate_systems.webp
+   :align: center
+   :alt: 3D coordinate systems comparison chart
+
+   Image by `Freya Holmér <https://twitter.com/FreyaHolmer>`__
+
+
+.. _doc_introduction_to_3d_space_and_manipulation:
+
+Space and manipulation gizmos
+-----------------------------
+
+Moving, rotating, and/or scaling objects in the 3D view is done through the
+manipulator gizmos.
+Each axis is represented by a color: Red, Green, Blue represent X, Y, Z
+respectively. This convention applies to the grid and other gizmos too
+(and also to the shader language, ordering of components for
+Vector3, Color, etc.).
+
+.. image:: img/tuto_3d5.webp
+
+Some useful keybindings:
+
+-  To snap placement or rotation, press :kbd:`Ctrl` while moving, scaling
+   or rotating.
+-  To center the view on the selected object, press :kbd:`F`.
+
+In the viewport, the arrows can be clicked and held to move the object on an axis.
+The arcs can be clicked and held to rotate the object.
+To lock one axis and move the object freely in the other two axes, the colored rectangles
+can be clicked, held, and dragged.
+
+If transform mode is changed from *Select Mode* to *Scale Mode*, the arrows will be 
+replaced by cubes, which can be dragged to scale an object as if the object is being moved.
+
+Navigating the 3D environment
+-----------------------------
+
+In 3D environments, it is often important to adjust the viewpoint or angle
+from which you are viewing the scene.
+In Godot, navigating the 3D environment in the viewport (or spatial editor)
+is possible in multiple ways.
+
+Default 3D scene navigation controls are similar to Blender (aiming to
+have some sort of consistency in the free software pipeline..), but
+options are included to customize mouse buttons and behavior to be
+similar to other tools in the Editor Settings. To change the controls
+to Maya or Modo controls, you can navigate to **Editor Settings > Editors > 3D**.
+Then, under *Navigation*, search for *Navigation Scheme*.
+
+.. image:: img/tuto_3d4.webp
+
+Using the default settings, the following shortcuts introduce how one can
+navigate in the viewport:
+
+Pressing :kbd:`Middle mouse button` and dragging the mouse allows you to orbit around 
+the center of what is on the screen.
+
+It is also possible to :kbd:`Left-click` and hold the manipulator gizmo located
+on the top right of the viewport to orbit around the center:
+
+.. image:: img/tuto_3d_gizmo.webp
+
+:kbd:`Left-clicking` on one of the colored circles will set the view to the chosen
+orthogonal and the current will be shown will be updated accordingly.
```

Comment:
This sentence is fragmentary

Reply:
"[...]and the viewport's view menu will be updated accordingly."